guides/v2.2/cloud/reference/discover-deploy.md

```diff
@@ -31,33 +29,101 @@ A set of YAML configuration files located in the project root directory define y
+
+-  **Use the same variables across environments**–The values for these variables may differ across environments, but the same variables are usually required in each environment. See [Configuration management for store settings](/{{ page.baseurl }}//cloud/live/sens-data-over.html).
+
+-  **Keep sensitive configuration values and data in environment-specific variables**–This includes variables specified using the Magento Cloud CLI, the Project Web interface, or added to the `env.php` file. See [Working with environment variables]({{ page.baseurl }}/cloud/env/working-with-variables.html).
+
+-  **Ensure that all code is available in the environment branch**–If you point to or include hooks to code in other branches, such as a private branch, you may encounter problems with the build and deploy process. For example, if you include a theme from a private repository in your branch, the theme will not build with the Magento code.
+
+-  **Add new extensions, integrations, and code in iterated branches**–Make and test changes locally, push to Integration, then to Staging and Production. Test and resolve issues in each environment before merging the updates to the next environment. Some extensions and integrations must be enabled and configured in a specific order due to dependencies. Adding these in groups can make your build and deploy process much easier and help determine where issues occur.
+
+-  **Verify service versions and relationships and ability to connect as needed**–See [Service versions]({{ page.baseurl }}/cloud/project/project-conf-files_services.html#service-versions) and [Service relationships]({{ page.baseurl }}/cloud/project/project-conf-files_services.html#service-relationships).
+
+-  **Check Production environment configuration**–Before deploying to Production, complete the following tasks:
+
+   -  Ensure that you can connect to all three nodes in the Production environment using [SSH]({{ page.baseurl }}/cloud/env/environments-ssh.html#ssh).
+
+   -  Verify that Indexers are set to _Update on Schedule_. See [Indexing modes]({{ page.baseurl }}/extension-dev-guide/indexing.html) in the _Extension Developer Guide_.
+
+   -  Run the {{ site.data.var.ece }} [smart wizard]({{ page.baseurl }}/cloud/deploy/smart-wizards.html) to ensure that your Cloud project configuration follows best practices for build and deployment.
+
+   -  Prepare the environment by updating any environment-specific variables in the Production code, verifying service availability and compatibility, and making any other required configuration changes.
```

Comment:
A lot of these seem like they are already pointed out earlier, but maybe it's okay to say them again.

Reply:
I think the issue is that people are not checking the Production environment before they deploy, so this is a reminder.